cordova-sdk/plugin.xml

```diff
@@ -53,16 +53,7 @@
             <!-- Optimove initProvider-->
             <provider android:authorities="${applicationId}.optimoveinitprovider" android:name="com.optimove.android.cordova.OptimoveInitProvider" android:exported="false" />
         </config-file>
-        <edit-config file="AndroidManifest.xml" target="/manifest" mode="merge">
-            <manifest xmlns:tools="http://schemas.android.com/tools" />
```

Comment:
We don't use `tools:` nodes for anything else? (I think no, just confirming)

Reply:
let's start by taking down edit-config removing backup rules